src/index.js

```diff
@@ -0,0 +1,337 @@
+    initializeCity();
+};
+
+function initMap() {
+    navigator.geolocation.getCurrentPosition(showMap, noPosition);
+};
+
+// if the city is not found then displays a map with coordinates
+function initializeCity() {
+    const isFahrenheit = localStorage.getItem('isFahrenheit') === 'true';
+    const units = isFahrenheit ? 'imperial' : 'metric';
+    const lang = localStorage.getItem("lang") || 'ru';
+    const city = localStorage.getItem('city');
+    if (city) {
+        showWeatherNow(city, lang, units);
+    } else {
+        initMap();
+    }
+};
+
+initializeCity();
+
+function showCoordinats(latitudeNow, longitudeNow) { // переименовать show
+    const lat = String(latitudeNow).split(".");
+    const lon = String(longitudeNow).split(".");
+    const latMinutes = lat[0];
+    const latSeconds = lat[1];
+    const lonMinutes = lon[0];
+    const lonSeconds = lon[1];
+
+    latitude.textContent = `${info.positions.latit} ${latMinutes}°  ${latSeconds}'`;
+    longitude.textContent = `${info.positions.longit} ${lonMinutes}°  ${lonSeconds}'`;
+    buttonSearch.textContent = `${info.search.but}`;
+    inputCity.placeholder = `${info.search.input}`;
+};
+
+function getMap(latitudeNow, longitudeNow) {
+    mapboxgl.accessToken = 'pk.eyJ1Ijoic3RhbHBldGMiLCJhIjoiY2trNWFqNmU4MDlhaDJvbGtocjlnN2s1ZiJ9.aDls4v3wiKMmvUBS76whKQ';
+    const map = new mapboxgl.Map({
+        container: 'map', // container id
+        style: 'mapbox://styles/mapbox/streets-v11', // style URL
+        center: [longitudeNow, latitudeNow], // starting position [lng, lat]
+        zoom: 10 // starting zoom
+    });
+    const marker = new mapboxgl.Marker()
+        .setLngLat([longitudeNow, latitudeNow])
+        .addTo(map);
+};
+
+async function getLinkToImage() {
+    const url = 'https://api.unsplash.com/photos/random?query=morning&client_id=njpPeASTxZr7ZpJtq-JokPEQFvKKFLwCXKqboncAt0Y';
+    const res = await fetch(url);
+    const data = await res.json();
+    return data.urls.regular;
+};
+
+async function getBackground() {
+    try {
+        const backgroundLink = await getLinkToImage();
+        body.style.backgroundImage = `url(${backgroundLink})`;
+        body.style.transition = "1s";
+    } catch (error) {
+        console.log(error);
+    }
+};
+
+// add inter
+function KeyBoard(e) {
+    if (e.which === 13) {
+        const city = inputCity.value;
+        const lang = localStorage.getItem("lang");
+        showSearchCity(city, lang);
+        getBackground()
+    }
+};
+
+buttonSearch.onclick = function (e) { //mouse click
+    if (e.which == 1) {
+        const city = inputCity.value;
+        const lang = localStorage.getItem("lang");
+        showSearchCity(city, lang);
+        getBackground()
+    }
+};
+
+window.addEventListener("keypress", KeyBoard);
+buttonRefresh.addEventListener("click", getBackground);
+buttonEnglishlanguage.addEventListener("click", langEn);
+buttonRussianLanguage.addEventListener("click", langRu);
```

Comment:
Please change to: 
buttonRussianLanguage.addEventListener("click", () => setLanguage('ru'));

Reply:
well done